src/components/Tabs/Tabs.js

```diff
@@ -33,7 +33,7 @@ export default class Tabs extends Component {
 	 * Update the childProps based on the updated properties passed to the card.
 	 */
 	componentWillReceiveProps(properties) {
-		const { style, initialSelectedIndex, ...childProps } = properties; // eslint-disable-line no-unused-vars
+		const { style, initialSelectedIndex, onChange, ...childProps } = properties; // eslint-disable-line no-unused-vars
```

Comment:
go ahead and remove the `label` prop as well since it does not been to be a property of the top level `div`